tutorial/libFuzzerTutorial.md

```diff
@@ -5,13 +5,13 @@ In this tutorial you will learn how to use [libFuzzer](http://libfuzzer.info)
  -- a coverage-guided in-process fuzzing engine.
 
 You will also learn basics of
-[AddressSanitizer](http://clang.llvm.org/docs/AddressSanitizer.html) --
+[AddressSanitizer](https://clang.llvm.org/docs/AddressSanitizer.html) --
 a dynamic memory error detector for C/C++.
 
 Prerequisites: experience with C/C++ and Unix shell.
 
 
-## Setup the environment
```

Comment:
It's a reasobale chance, but it will invalidate the link to the subsection, so maybe not...